doc/adr/0006-no-certificat-pinning.md

```diff
@@ -0,0 +1,28 @@
+# 5. No certificate pinning
+
+Date: 2022-12-01
+
+## Status
+
+Accepted
+
+## Context
+
+During the identification flow, the eID-Client connects to the server of a Service Provider (eService) and 
```

Comment:
Is it really called 'eService'?

Reply:
"eService" is the term for Service Provider (Diensteanbieter) which is used in the technical guidelines.